bookmark-service/src/main/java/fr/loicmathieu/bookmarkit/BookmarkResource.java

```diff
@@ -25,52 +28,57 @@
 
     @GET
     @Operation(summary = "List all bookmarks")
-    @Counted(name = "listAll.count")
-    @Timed(name="listAll.time")
-    public List<Bookmark> listAll(){
+    @Counted(name = "listBookmarks.count")
+    @Timed(name = "listBookmarks.time")
+    public List<Bookmark> listBookmarks() {
         return Bookmark.listAll();
     }
 
     @GET
-    @Path("/{id}")
+    @Path("{id}")
     @Operation(summary = "Get a bookmark")
-    @Counted(name = "get.count")
-    @Timed(name="get.time")
-    public Bookmark get(@PathParam("id") Long id) {
+    @Counted(name = "getBookmark.count")
+    @Timed(name = "getBookmark.time")
+    public Bookmark getBookmark(@PathParam("id") Long id) {
         return Bookmark.findById(id);
     }
 
     @POST
     @Transactional
     @Operation(summary = "Create a bookmark")
-    @Counted(name = "create.count")
-    @Timed(name="create.time")
-    public Response create(Bookmark bookmark){
-        bookmark.persist();
-        return Response.created(URI.create("/bookmarks/" + bookmark.id)).build();
+    @Counted(name = "createBookmark.count")
+    @Timed(name = "createBookmark.time")
+    public Response createBookmark(@Valid Bookmark bookmark) {
```

Comment:
Bean validation is not part of this DOJO.
Can you remove the `@Valid` annotation ?